src/impi/core.cljs

```diff
@@ -231,10 +231,23 @@
 (defmulti create
   (fn [attr value] attr))
 
+(def ^:private renderer-arguments  
```

Comment:
I think maybe we could make this code a little neater by factoring out the renderer options. Also `renderer-arguments` would be more accurately named `renderer-create-keys`, because those are the keys that are dealt with when the renderer is created.

```clojure
(def ^:private renderer-create-keys
  [:pixi.renderer/size
   :pixi.renderer/antialias?
   :pixi.renderer/transparent?
   :pixi.renderer/no-webgl?
   :pixi.renderer/resolution])

(defn- renderer-options [options]
  #js {:transparent (options :transparent? false)
       :antialias   (options :antialias? false)
       :resolution  (options :resolution 1)})

(defmethod create :pixi/renderer
  [_ {[w h] :pixi.renderer/size :keys [pixi.renderer/no-webgl?] :as options}]
  {:val (select-keys options renderer-create-keys)
   :obj (js/PIXI.autoDetectRenderer w h (renderer-options options) no-webgl?)})
```